doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -5,28 +5,65 @@
 
 ## Table of Contents
 * [3.1 Introduction](#3.1)
-* [3.2 Measurement Criteria](#3.2)
-  * [3.2.1 Storage and IOPS](#3.2.1)
-* [3.3 Measurement Types](#3.3)
-  * [3.3.1 Performance Measurements](#3.3.1)
-  * [3.3.2 Resiliency Measurements](#3.3.2)
-* [3.4 NFVI Test Cases](#3.4)
+* [3.2 Assumptions](#3.2)
+* [3.3 Requirement Type](#3.3)
+* [3.4 Profile Catalog](#3.4)
+* [3.5 Software & Hardware Reference](#3.5)
+* [3.6 Options & Extensions](#3.6)
+* [3.7 Measurement Criteria](#3.7)
+  * [3.7.1 Storage and IOPS](#3.7.1)
+* [3.8 Measurement Types](#3.8)
+  * [3.8.1 Performance Measurements](#3.8.1)
+  * [3.8.2 Resiliency Measurements](#3.8.2)
+* [3.9 NFVI Test Cases](#3.9)
 
 <a name="3.1"></a>
 ## 3.1 Introduction
 
 > Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
 
 <a name="3.2"></a>
-## 3.2 Measurement Criteria
+## 3.2 Assumptions
+
+Content to be written:
+- Assumptions:  Automatable, Integrated with CICD tool chain
```

Comment:
We should refer to the generic requirements listes in RM ch 8.2.4 and only add new ones specific to the LFN compliance program - even though I am currently not clear on whether or not there are specific ones.

Reply:
Agree @gkunz .